### PR TITLE
[Keymap] Updates to Personal Kyria Keymap

### PR DESCRIPTION
**Pull request by john-ezra**
Major changes to OLED displays.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/splitkb/kyria/keymaps/john-ezra/readme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### This Keymap uses the nanoBoot bootloader to maximize available firmware space.
+
+As is, the firmware will be too large if using the standard bootloader.  Removing `DEBOUNCE_TYPE = sym_eager_pk` from the `rules.mk` should get the size under the limit.
+
+If nanoBoot is not going to be used, delete or comment out the following from your `rules.mk`:
+```BOOTLOADER = hid
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/john-ezra/readme.md:6]:
```suggestion
```make
BOOTLOADER = hid
```

File: keyboards/splitkb/kyria/keymaps/john-ezra/glcdfont.c [hunk 2]
```diff
@@ -0,0 +1,260 @@
+#include "progmem.h"
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/john-ezra/glcdfont.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/splitkb/kyria/keymaps/john-ezra/keymap.c [hunk 3]
```diff
@@ -167,6 +219,14 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
         }
       }
       break;
+    case MSS_CTL:
+      if (record->tap.count && record->event.pressed) {
+          tap_code16(C(KC_UP));
+          return false;
+      } else if (record->event.pressed) {
+          return true;
+      }
```
Comment (fauxpark) [inline keyboards/splitkb/kyria/keymaps/john-ezra/keymap.c:228]:
```suggestion
      if (record->event.pressed &&record->tap.count) {
          tap_code16(C(KC_UP));
          return false;
      }
```
Can be simplified a little.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**